docs/eager.md

```diff
@@ -0,0 +1,122 @@
+# Eager Mode + Compile API
+
+In this doc we will go over how to use PyTorch/XLA’s new experimental `eager` mode with the `compile` API. The goal is to make PyTorch/XLA experience more aligned with the native PyTorch and make development process easier.
+
+
+## Background
+Currently PyTorch/XLA runs on the LazyTensor tracing mode by default. In the following code
+```python
+import torch
+import torch_xla
+import torchvision
+
+device = torch_xla.device()
+model = torchvision.models.resnet18().to(device)
+input = torch.randn(64, 3, 224, 224).to(device)
+
+# model tracing
+res = model(input)
+
+# model execution, you can also use `xm.mark_step`
```

Comment:
Do you think we should be mentioning `mark_step` in new documentation? Ultimately we should deprecated it IMO

Reply:
my thinking is most people don't know what `torch_xla.sync` is at this point. We should at least mention it so they know `mark_step` and `sync` are the same thing.